app/models/ability.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    if user.has_role?(:admin)
+      can :manage, :all
+    else
+      can %i[read create delete], Article if user.has_role?(:author, Article)
```

Comment:
Let's talk about this condition tomorrow. Can you please remind me about this tomorrow?

Reply:
sure i will